spec/chatroom.js

```diff
@@ -3326,6 +3327,54 @@
                 expect(info_msgs.pop().textContent).toBe("annoyingGuy has been given a voice again");
                 done();
             }));
+
+            it("takes /destroy to destroy a muc",
+                mock.initConverseWithPromises(
+                    null, ['rosterGroupsFetched'], {},
+                    async function (done, _converse) {
+
+                await test_utils.openAndEnterChatRoom(_converse, 'lounge', 'localhost', 'dummy');
+                var sent_IQ, IQ_id;
+                var sendIQ = _converse.connection.sendIQ;
+                spyOn(_converse.connection, 'sendIQ').and.callFake(function (iq, callback, errback) {
+                    sent_IQ = iq;
+                    IQ_id = sendIQ.bind(this)(iq, callback, errback);
+                });
```

Comment:
Not a dealbreaker for merging, but I'd like us to move away from using `var` in favour of `let` and `const`.